### PR TITLE
removed mongo connection in tests

### PR DESCRIPTION
**Pull request by 8rianHo**
deleted files as a mongodb connection is no longer needed for tests 
- jest.config.js
- jestSetup.js
- env.test.js

created a users fixtures file for dummy data

updated test files to use jest.spyOn with jest.restoreAllMocks called in the afterEach block

### CODE REVIEW

File: tests/users.test.js [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
+const Users = require("../models/users");
+const { userOne } = require("./fixtures/users");
 
-afterAll(async () => {
-  await mongoose.connection.close();
+afterEach(() => {
+  jest.restoreAllMocks();
 });
 
 test("should get users", async () => {
```
Comment (lujasmine) [inline tests/users.test.js:10]: Would be good to have `describe` wrappers for each route, as there will probably be more than one test per route eventually

```suggestion
describe("GET /users", () => {
  test("should get users", async () => {
  ...
```

File: tests/users.test.js [hunk 2]
```diff
@@ -1,20 +1,21 @@
-const request = require("supertest");
 const app = require("../app");
-const mongoose = require("mongoose");
-const { setupTestDatabase } = require("../tests/fixtures/db");
-
-beforeEach(setupTestDatabase);
+const request = require("supertest");
+const Users = require("../models/users");
+const { userOne } = require("./fixtures/users");
```
Comment (lujasmine) [inline tests/users.test.js:4]: Nice 💯

**Review by lujasmine — APPROVED** [review]
Comment is non blocking ✅

File: tests/users.test.js (hunk 1 above)
Comment (8rianHo) [reply tests/users.test.js:10]: good idea!

File: tests/users.test.js [hunk 3]
```diff
@@ -1,23 +1,32 @@
...
 });
 
-test("should create a new user", async () => {
-  const response = await request(app)
-    .post("/users")
-    .send({ message: "i am new" })
-    .expect(201);
-  expect(response.text).toEqual("created new user");
+describe("POST /users", () => {
+  afterEach(() => {
```
Comment (lujasmine) [inline tests/users.test.js:20]: Just looking at this, potentially could refactor even more, by having an overall `describe` wrapper, and only have the `afterEach` once, as it is currently the same for both

**Pull request merged**